c/CMakeLists.txt

```diff
@@ -258,32 +259,19 @@ if(BLAKE3_USE_TBB)
     set(APPEND BLAKE3_CXX_STANDARD_FLAGS_GNU -std=c++20)
     set(APPEND BLAKE3_CXX_STANDARD_FLAGS_MSVC /std:c++20)
   endif()
+
   set(BLAKE3_CXXFLAGS_GNU "-fno-exceptions;-fno-rtti;${BLAKE3_CXX_STANDARD_FLAGS_GNU}" CACHE STRING "C++ flags used for compiling private BLAKE3 library components with GNU-like compiler frontends.")
   set(BLAKE3_CXXFLAGS_MSVC "/EHs-c-;/GR-;${BLAKE3_CXX_STANDARD_FLAGS_MSVC}" CACHE STRING "C++ flags used for compiling private BLAKE3 library components with MSVC-like compiler frontends.")
```

Comment:
I've been thinking on whether it might make sense to merge the cache variables into a single `BLAKE3_CXXFLAGS` with a default based on `BLAKE3_CXX_COMPILER_FRONTEND_VARIANT`, but haven't had time to experiment.

Reply:
What is the intended use case?

Why don't we do that in a separate PR just to keep things focused?